src/cdk/collections/collections.md

```diff
@@ -1,3 +1,34 @@
-### Collections
+The `collections` package provides a set of utilities for managing collections.
 
-A set of utilities for managing collections.
\ No newline at end of file
+### `SelectionModel`
+`SelectionModel` is a utility for powering selection of one or more options from a list.
+This utility is used in components such as the selection list, table selections and chip lists.
```

Comment:
Something like
```markdown
`SelectionModel` stores the selection state for components that support multiple selection.
```

The word "utility" isn't a good fit here

Reply:
How about `model`?

Reply:
That works